src/components/BulkEditList/BulkEditListFilters/BulkEditListFilters.js

```diff
@@ -57,37 +67,91 @@ export const BulkEditListFilters = () => {
     setDropZoneActive(false);
   };
 
-  const handleDrop = () => {
-    setLoading(false);
+
+  const handleChange = (event, field) => setFilters(prev => ({
+    ...prev, [field]: event.target.value,
+  }));
+
+  const hanldeCapabilityChange = (event) => setFilters(prev => ({
+    ...prev, capabilities: event.values,
+  }));
+
+  const uploadFileFlow = async (fileToUpload) => {
+    setFileUploadedName(fileToUpload.name);
     setDropZoneActive(false);
+
+    try {
+      const { id } = await requestJobId({ recordIdentifier });
+
+      await fileUpload({ id, fileToUpload });
+
+      history.replace({
+        pathname: `/bulk-edit/${id}`,
+        search: `?fileName=${fileToUpload.name}`,
+      });
+
+      setIsFileUploaded(true);
+    } catch ({ message }) {
+      showCallout({ message });
```

Comment:
could you please add error message to translations and use it instead of e.message?

Reply:
Done. I add an error message on upload.